### PR TITLE
App config for admins - recommendation for devs to document it

### PR DESCRIPTION
**Pull request by ashishguptaiitb**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/promote-app-adoption.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4b8fb84cf8b3482321f0166b5853e9624f360a52/msteams-platform%2Fpromote-app-adoption.md) | 95 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8a2afc46-3a2a-4f8c-b39a-bc57bc4a9460) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/promote-app-adoption.md](https://github.com/MicrosoftDocs/msteams-docs/blob/989032ca78b669e8fefdc0542983b853412eba2c/msteams-platform%2Fpromote-app-adoption.md) | 91 | 50 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7a1952a3-4be4-4c5c-b723-2dce34369198) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/promote-app-adoption.md](https://github.com/MicrosoftDocs/msteams-docs/blob/84d7a2c578e65b1915d9d446b62220a49396652b/msteams-platform%2Fpromote-app-adoption.md) | 91 | 50 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e844db5c-5047-4c51-954d-1dee637c7d75) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/promote-app-adoption.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e04d3ec8be2c48abd31d18d13085618a3cee2cc8/msteams-platform%2Fpromote-app-adoption.md) | 92 | 48 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e869d593-9d00-4d38-a149-480b2c43e1ad) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/promote-app-adoption.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Admins can set up an app governance process that manages your organization's IT
 
 ## Understand how you can drive app adoption
 
-Following the below five easy steps, you can encourage customers to adopt your apps through admin recommendations to their users, for maximum adoption in an organization. Consider the following elements for app adoption.
+By following these five easy steps, you make it easy for your customers to maximum adoption within their organizations.
```
Comment (v-ypalikila) [inline msteams-platform/promote-app-adoption.md:42]: Can we re-write it as "By following these five easy steps, you make it easy for your customers to maximize app adoption within their organizations."

File: msteams-platform/promote-app-adoption.md [hunk 2]
```diff
@@ -297,10 +297,11 @@ When your app is being rolled out or being used, your customers might have queri
...
 
-In your app’s help documentation, consider providing some or all the following information to provide reliable support to your customers and admins:
+In your app’s help documentation, consider providing some or all the following information for your customers and admins:
 
-* Contact method with support information, for example, email, phone, or web portal.
-* Links to help documentation about rollout for admins, features for users, and troubleshooting to supporting functions.
+* Contact method such as email, phone, community forum, or web portal for customers who want to contact you for app support.
+* Explain features for users, app rollout and configuration methods for admins, and troubleshooting to supporting functions.
+* Explain the different personas that are required to configure your app. For example, an app that works with a CRM or a bug tracking system may require an admin to provision accounts for app users or to create an account for the app to fetch your organization's information.
```
Comment (v-ypalikila) [inline msteams-platform/promote-app-adoption.md:304]: Can we expand the abbreviation CRM. Is it Customer Relationship Management?

**Review by v-ypalikila — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/promote-app-adoption.md](https://github.com/MicrosoftDocs/msteams-docs/blob/009bb2b300f13bc1f496a470d30dd382039fdabf/msteams-platform%2Fpromote-app-adoption.md) | 92 | 48 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6516b4ac-f1ad-4e81-8528-3d6501991f66) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/promote-app-adoption.md (hunk 1 above)
Comment (ashishguptaiitb) [reply msteams-platform/promote-app-adoption.md]: Updated.

File: msteams-platform/promote-app-adoption.md (hunk 2 above)
Comment (ashishguptaiitb) [reply msteams-platform/promote-app-adoption.md]: Updated.

**Review by surbhigupta12 — APPROVED** [review]

**Review by v-ypalikila — APPROVED** [review]
Approved.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/promote-app-adoption.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6e0b33fa9713f151c6359681fa4c137bef1082bb/msteams-platform%2Fpromote-app-adoption.md) | 92 | 48 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1f6122fc-56bd-4c98-b9c2-8f64e6a3f979) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**